storage/tianmu/types/tianmu_data_types.h

```diff
@@ -301,6 +301,8 @@ class TianmuDateTime : public ValueBasic<TianmuDateTime> {
   TianmuDateTime(int64_t dt, common::CT at);
```

Comment:
`CT` can be `ColumnType`, it can be in an other PR.

Reply:
Fixed it.